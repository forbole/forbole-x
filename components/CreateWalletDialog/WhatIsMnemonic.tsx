import { Box, DialogContent, Typography } from '@material-ui/core'
import useTranslation from 'next-translate/useTranslation'
import React from 'react'
import useStyles from './styles'

const WhatIsMnemonic: React.FC = () => {
  const { t } = useTranslation('common')
  const classes = useStyles()

  return (
    <DialogContent className={classes.dialogContent}>
      <Box mx={1} mb={4}>
        <Typography paragraph>{t('what is secret recovery phrase description')}</Typography>
      </Box>
    </DialogContent>
  )
}

export default WhatIsMnemonic
