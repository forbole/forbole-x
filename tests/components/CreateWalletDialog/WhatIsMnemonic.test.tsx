import React from 'react';
import renderer from 'react-test-renderer';
import WhatIsMnemonic from '../../../components/CreateWalletDialog/WhatIsMnemonic';

describe('component: CreateWalletDialog - WhatIsMnemonic', () => {
  it('renders correctly', () => {
    const component = renderer.create(<WhatIsMnemonic />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

afterEach(() => {
  jest.clearAllMocks();
});
