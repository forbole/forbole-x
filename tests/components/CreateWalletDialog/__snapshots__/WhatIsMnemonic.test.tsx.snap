// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component: CreateWalletDialog - WhatIsMnemonic renders correctly 1`] = `
<div
  className="MuiDialogContent-root HookGlobalStyles-dialogContent-7"
>
  <div
    className="MuiBox-root MuiBox-root-8"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
    >
      common:what is mnemonic description
    </p>
  </div>
</div>
`;
